x-pack/plugins/profiling/public/views/add_data_view/index.tsx

```diff
@@ -66,193 +79,203 @@ export function AddDataView() {
   const symbolUrl = data?.symbolizer?.host;
   const stackVersion = data?.stackVersion!;
   const majorVersion = stackVersion ? major(stackVersion).toString() : undefined;
+  const supportedCPUArchitectures = ['x86_64', 'arm64'];
```

Comment:
@inge4pres this [check](https://buildkite.com/elastic/kibana-pull-request/builds/169473#018b482b-be95-4ac4-a8a4-b073448cf0e1/238-247) is failing because you defined the `supportedCPUArchitectures` inside the React component. If you move it to after line 57 before 
```js
export function AddDataView() {
```
The error will be solved.

Reply:
ahhh thanks, indeed i was thinking to ask you about the error 🙏🏼 thanks, will move it